### PR TITLE
Migrates to Bootstrap 5, closes #266

### LINKED ISSUE #266: Bootstrap 3 deprecation
Hello!

It seems [bootstrap 3 is deprecated](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1088438) and this means it probably won't be in the next Debian release...

Bootstrap v5 apparently isn't a drop-in replacement, so migrating to it would require some work (I haven't really looked...).

Happy to run some tests if you need me to!

https://getbootstrap.com/docs/5.3/migration/

### PR DESCRIPTION
**Pull request by ogarcia**
If I have not forgotten anything with this change Bootstrap is updated to the latest version (right now the 5.3.3). As with the previous version both CSS and JS is embedded in the static directory so you don't have to go looking for these files anywhere.

Before merging this PR it would be best to do some tests in case something goes wrong. I have tested with Firefox and Chrome and everything is fine, but four eyes see more than two.

I have found a bug that is not due to this change (it happens with the current version) and it is that it is not possible to change the email address. From what I have seen it seems to me that the function that handles the post to `changemail` is not complete, it picks up the address that is passed by the form but does nothing with it.

Of course, any comments are welcome.

### CODE REVIEW

**Comment by ogarcia** [comment]
Holy shit, the unit tests! Tomorrow I'll take a look at them, it will be something silly. :stuck_out_tongue_winking_eye:

**Comment by ogarcia** [comment]
From my point of view I think I have corrected everything and this PR is ready to merge. If there are any problems, let me know.

File: supysonic/static/img/vinyl.svg
Comment (baldurmen) [inline supysonic/static/img/vinyl.svg:1]: Are you the author of that file? If not, it would be good to make sure to mention its license somewhere

Comment (ogarcia) [reply supysonic/static/img/vinyl.svg:1]: It is [one of the Bootstrap icons](https://icons.getbootstrap.com/icons/vinyl-fill/) (like the rest of the icons on the page).

**Pull request merged**